ui/components/app/transaction-list-item/index.scss

```diff
@@ -58,7 +58,7 @@
   }
 
   .transaction-status-label::after {
-    content: "·";
+    content: '·';
```

Comment:
Why this change?

Reply:
I thinks it's Albert local prettier doing this on it's own 😅 
@albertolive can you check?